lib/kbsecret/cli/kbsecret-rm

```diff
@@ -12,29 +12,33 @@ cmd = CLI.create do |c|
   c.slop do |o|
     o.banner = <<~HELP
       Usage:
-        kbsecret rm [options] <record>
+        kbsecret rm [options] <record [records ...]>
```

Comment:
Depluralize `records`, for consistency with `kbsecret env` and `kbsecret login`:

```
kbsecret rm [options] <record [record ...]>
```